src/layouts/ResourceRoom/ResourceRoom.tsx

```diff
@@ -96,7 +116,10 @@ const EmptyResourceRoom = () => {
           >
             <Input
               maxW="50%"
-              {...register("newDirectoryName", { required: true })}
+              {...register("newDirectoryName", {
+                required:
+                  "Please ensure that you have entered a resource room name!",
+              })}
```

Comment:
Actually, could we have the default value of this field be the existing resource room name?

Reply:
that sounds fine - so for clarity's sake, this means that there will be **no** placeholder and instead, the field will be prefilled w/ the existing resource room name right?

@NatMaeTan wdyt about this change? i think it's reasonable

Reply:
Yup, I think this is more consistent with the existing renaming behaviour of other items (e.g. folders/files come prefilled with the existing name)

Reply:
This is for changing a resource room name right? If so, yup sounds good that it should be prefilled with the existing name for people to edit

Reply:
done!